### ISSUE TITLE
🚧 🚀 Review Section - Elementor

### ISSUE THREAD

**Issue by lewishomerenovations** [elementor/elementor#20593]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I am currently working on my clients website and there is an error within the 'Reviews' plug-in. The size of the review section does not change regardless of the size of the review, which makes the section look disproportionate. Is there a way we can make the review section change size to match the review in focus. I have attached a visual aid so you can view the issue.
[Review Section Errors - LHR - For Elementor - 03-11 (1).pdf](https://github.com/elementor/elementor/files/10140596/Review.Section.Errors.-.LHR.-.For.Elementor.-.03-11.1.pdf)

### Describe the solution you'd like

It would be a massive help if this plug-in could be assessed and reformatted, so that each review was proportionate to the overall section. For example, a one line review, should have the relative space for a review of it's size, rather than adhere to the spacing for the largest review inputted.

**Comment by nicholaszein** [comment]
Hello @lewishomerenovations,

Thanks for your feedback! 🙏 

### 🧐 While reviewing your request we noticed it was missing the following key information:

- **Clear and concise title:** You did not write a clear and concise title;
- **Prerequisites:** Please search the repository before submitting an issue;

I also am not sure if I follow as well. Are you talking about [this widget](https://elementor.com/help/reviews-widget-pro/)? Or is it another plugin you've installed through your WordPress dashboard?

Please try to be as detailed as possible so we can help you better. 🙏 

Kind regards

**Comment by nicholaszein** [comment]
Hello there @lewishomerenovations, 

#### 🕙 It's been a while since we answered you, and we haven't received any feedback. 

## 🚧 For this reason, we are closing this issue because of no activity.

If you think the widget in question was the one I asked ([the Reviews widget](https://elementor.com/help/reviews-widget-pro/)), try using the following Custom CSS:

````css

selector .swiper-slide {
    display: flex;
    flex-direction: column;
    justify-content: center;
}
````

👉 If you wish to continue this discussion, please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

👉 If you wish to continue this discussion, please follow the guidelines for opening a new [🚀 Feature Request](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=feature_request.yml).

Feel free to submit a new issue in accordance with these guidelines. 😉 

Kind regards
